### PR TITLE
Add Packaging for hammer_cli_foreman_salt (RPM)

### PR DESCRIPTION
**Pull request by stbenjam**
Don't merge just yet, needs foreman_salt 2.0 to be released first.

### CODE REVIEW

File: rubygem-hammer_cli_foreman_salt/foreman_salt.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+:foreman_salt:
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_salt/foreman_salt.yml]: You should add this to the next gem release, then we can copy it out instead.

File: rubygem-hammer_cli_foreman_salt/rubygem-hammer_cli_foreman_salt.spec [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+%files
+%dir %{gem_instdir}
+%{gem_instdir}/lib
+%config(noreplace) %{_sysconfdir}/%{confdir}/cli.modules.d/foreman_salt.yml
+%exclude %{gem_cache}
+%{gem_spec}
+
+%files doc
+%doc %{gem_dir}/doc/%{gem_name}-%{version}
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_salt/rubygem-hammer_cli_foreman_salt.spec]: %{gem_docdir}

File: rubygem-hammer_cli_foreman_salt/foreman_salt.yml (hunk 1 above)
Comment (mmoll) [inline rubygem-hammer_cli_foreman_salt/foreman_salt.yml]: 0.0.2 of the gem is released now with the config file included

**Comment by domcleal** [comment]
@stbenjam please rebase when you get the opportunity

File: rubygem-hammer_cli_foreman_salt/rubygem-hammer_cli_foreman_salt.spec [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/README.md
+
+%changelog
+* Tue Mar 03 2015 Stephen Benjamin <stephen@redhat.com> 0.0.3-1
+- Release 0.0.3
+
+* Wed Jan 15 2015 Stephen Benjamin <stephen@redhat.com> 0.0.1-1
```
Comment (domcleal) [inline rubygem-hammer_cli_foreman_salt/rubygem-hammer_cli_foreman_salt.spec]: Nitpick to fix RPM warnings: Sunday, or the 11th/15th Jan

**Comment by stbenjam** [comment]
@domcleal updated

File: rubygem-hammer_cli_foreman_salt/rubygem-hammer_cli_foreman_salt.spec (hunk 3 above)
Comment (domcleal) [inline rubygem-hammer_cli_foreman_salt/rubygem-hammer_cli_foreman_salt.spec]: oops, was looking at Feb :)

**Comment by domcleal** [comment]
Thanks @stbenjam!  Will build for nightlies and 1.8.

**Pull request merged**